.jenkins/pytorch/build.sh

```diff
@@ -167,3 +173,27 @@ if [[ "$BUILD_TEST_LIBTORCH" == "1" ]]; then
   make VERBOSE=1
   popd
 fi
+
+# Test XLA build
+if [[ "${JOB_BASE_NAME}" == *xla* ]]; then
+  # TODO: Move this to Dockerfile.
+  # Bazel dependencies
+  sudo apt-get -qq update
+  sudo apt-get -qq install pkg-config zip zlib1g-dev unzip
+  # XLA build requires Bazel
+  wget https://github.com/bazelbuild/bazel/releases/download/0.21.0/bazel-0.21.0-installer-linux-x86_64.sh
+  chmod +x bazel-*.sh
+  ./bazel-*.sh --user
+  export PATH="$PATH:$HOME/bin"
+  BAZEL="$(which bazel)"
+  if [ -z "${BAZEL}" ]; then
+    echo "Unable to find bazel..."
+    exit 1
+  fi
+
+  # Bazel doesn't work with sccache gcc. https://github.com/bazelbuild/bazel/issues/3642
+  export CC=/usr/bin/gcc CXX=/usr/bin/g++
```

Comment:
Would this mean that it will always be a cold build? (I noticed that the build phase is taking 2h46m, we should probably find a way to make it work with sccache if possible)

Reply:
@ailzhang Bazel cloud build is probably something we can ask the Google team about